spec/e2e/ssh

```diff
@@ -0,0 +1,27 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Usually this isn't safe to keep in repos, but I generated this key just for e2e purposes.